### PR TITLE
Feuer Frei! Ship to ship weapons for third party ships

### PR DESCRIPTION
**Pull request by TheDanseMacabre**
Gives some third party ships ship to ship weapons, and buffs the small arms of a few more.

### CODE REVIEW

**Comment by TheDanseMacabre** [comment]
!review

**Review by Alberyk — APPROVED** [review]

File: html/changelogs/dansemacabre-feuerfrei.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - rscadd: "The Elyran strike craft, FSF and SFA corvettes, and TCFL corvette now all have grauwolf flak batteries. The militia ship has a 40mm rotary cannon. The civilian station has a large amount of ammunition to sell."
```
Comment (SleepyGemmy) [inline html/changelogs/dansemacabre-feuerfrei.yml:41]:
```suggestion
  - maptweak: "The Elyran strike craft, FSF and SFA corvettes, and TCFL corvette now all have grauwolf flak batteries. The militia ship has a 40mm rotary cannon. The civilian station has a large amount of ammunition to sell."
```

**Review by SleepyGemmy — COMMENTED** [review]
image
stairwell goes right into the weapon (both SFA and FSF).

image
not sure how i feel about the militia ship having a 7.62 assault rifle.

image
ditto (for both SFA and FSF).

image
ditto but 5.56.

**Comment by TheDanseMacabre** [comment]
> stairwell goes right into the weapon (both SFA and FSF).
That's fine
>people have rifles now
The increasing frequency with which other third party ships are equipped with rifles means that these factions need to have rifles of their own in order to have parity with other factions. I don't think it's fair for these factions to run around outgunned, and in any case the fact that rifles are being nerfed across the board and because of how we see third party ships act ingame makes it unlikely this is going to cause any practical problems, combined with the relatively limited quantity of ammunition per rifle.

**Comment by SleepyGemmy** [comment]
> The increasing frequency with which [...]

fair.

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]
image
fix the SFA and FSF staircase.

**Review by SleepyGemmy — APPROVED** [review]
requested changes addressed.

**Pull request merged**